### ISSUE TITLE
`sri_static` cannot find source file for hashing

### ISSUE THREAD

**Issue by hugorodgerbrown** [RealOrangeOne/django-sri#70]
We have an issue in production where the `sri_static` cannot find files on disk. On further inspection this appears to occur when we have run `collectstatic` to move files from `STATICFILES_DIRS` to `STATIC_ROOT` and then deleted the source files, which we do in production environments.

1. Edit source JS in `/frontend/src`
1. Run `yarn build` to process frontend app into `/frontend/dist` (`STATICFILES_DIRS`)
1. Run `collectstatic` to copy files to `/frontend/staticfiles` (`STATIC_ROOT`)**
1. Delete `/frontend/dist`

In this situation, at runtime, `sri.utils.get_static_path` will not find a source file, as it is using the `django.contrib.staticfiles.finders.find` function, which uses the `STATICFILES_FINDERS`, which are looking for source files, not the collected static output.

When running in product the finder should be looking in `STATIC_ROOT`, not `STATICFILES_DIRS`?

---

** Although we build the JS frontend into /dist, we need to scoop up all of the other app assets (e.g. admin), so we have to run `collectstatic` after we build the JS.

**Comment by hugorodgerbrown** [comment]
Steps to reproduce:

1. Run `collectstatic` locally to copy files into `tests/collect-static`

```
$ export DJANGO_SETTINGS_MODULE=tests.settings
$ django-admin collectstatic --noinput

130 static files copied to 'django-sri/tests/collected-static'.
```

2. Remove the source directory `tests/static`

```
$ mv tests/static tests/static2
```

3. Run the test suite.

```
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================== short test summary info ===================================================
FAILED tests/tests.py::test_simple_template - FileNotFoundError: index.js
FAILED tests/tests.py::test_complex_template - FileNotFoundError: index.js
FAILED tests/tests.py::test_algorithms_template - FileNotFoundError: index.js
FAILED tests/tests.py::test_generic_algorithm[index.css-Algorithm.SHA256] - FileNotFoundError: index.css
FAILED tests/tests.py::test_generic_algorithm[index.css-Algorithm.SHA384] - FileNotFoundError: index.css
FAILED tests/tests.py::test_generic_algorithm[index.css-Algorithm.SHA512] - FileNotFoundError: index.css
FAILED tests/tests.py::test_generic_algorithm[index.js-Algorithm.SHA256] - FileNotFoundError: index.js
FAILED tests/tests.py::test_generic_algorithm[index.js-Algorithm.SHA384] - FileNotFoundError: index.js
FAILED tests/tests.py::test_generic_algorithm[index.js-Algorithm.SHA512] - FileNotFoundError: index.js
FAILED tests/tests.py::test_default_algorithm - FileNotFoundError: index.js
FAILED tests/tests.py::test_get_static_path[index.css] - FileNotFoundError: index.css
FAILED tests/tests.py::test_get_static_path[index.js] - FileNotFoundError: index.js
FAILED tests/tests.py::test_hashes_are_consistent[index.css-Algorithm.SHA256] - FileNotFoundError: index.css
FAILED tests/tests.py::test_hashes_are_consistent[index.css-Algorithm.SHA384] - FileNotFoundError: index.css
FAILED tests/tests.py::test_hashes_are_consistent[index.css-Algorithm.SHA512] - FileNotFoundError: index.css
FAILED tests/tests.py::test_hashes_are_consistent[index.js-Algorithm.SHA256] - FileNotFoundError: index.js
FAILED tests/tests.py::test_hashes_are_consistent[index.js-Algorithm.SHA384] - FileNotFoundError: index.js
FAILED tests/tests.py::test_hashes_are_consistent[index.js-Algorithm.SHA512] - FileNotFoundError: index.js
FAILED tests/tests.py::test_integrity[index.css-Algorithm.SHA256] - FileNotFoundError: index.css
FAILED tests/tests.py::test_integrity[index.css-Algorithm.SHA384] - FileNotFoundError: index.css
FAILED tests/tests.py::test_integrity[index.css-Algorithm.SHA512] - FileNotFoundError: index.css
FAILED tests/tests.py::test_integrity[index.js-Algorithm.SHA256] - FileNotFoundError: index.js
FAILED tests/tests.py::test_integrity[index.js-Algorithm.SHA384] - FileNotFoundError: index.js
FAILED tests/tests.py::test_integrity[index.js-Algorithm.SHA512] - FileNotFoundError: index.js
FAILED tests/tests.py::test_sri_integrity_static[index.css-Algorithm.SHA256] - FileNotFoundError: index.css
FAILED tests/tests.py::test_sri_integrity_static[index.css-Algorithm.SHA384] - FileNotFoundError: index.css
FAILED tests/tests.py::test_sri_integrity_static[index.css-Algorithm.SHA512] - FileNotFoundError: index.css
FAILED tests/tests.py::test_sri_integrity_static[index.js-Algorithm.SHA256] - FileNotFoundError: index.js
FAILED tests/tests.py::test_sri_integrity_static[index.js-Algorithm.SHA384] - FileNotFoundError: index.js
FAILED tests/tests.py::test_sri_integrity_static[index.js-Algorithm.SHA512] - FileNotFoundError: index.js
FAILED tests/tests.py::test_caches_hash[index.css-Algorithm.SHA256] - FileNotFoundError: index.css
FAILED tests/tests.py::test_caches_hash[index.css-Algorithm.SHA384] - FileNotFoundError: index.css
FAILED tests/tests.py::test_caches_hash[index.css-Algorithm.SHA512] - FileNotFoundError: index.css
FAILED tests/tests.py::test_caches_hash[index.js-Algorithm.SHA256] - FileNotFoundError: index.js
FAILED tests/tests.py::test_caches_hash[index.js-Algorithm.SHA384] - FileNotFoundError: index.js
FAILED tests/tests.py::test_caches_hash[index.js-Algorithm.SHA512] - FileNotFoundError: index.js
FAILED tests/tests.py::test_manifest_storage[index.css] - ValueError: Missing staticfiles manifest entry for 'index.css'
FAILED tests/tests.py::test_manifest_storage[index.js] - ValueError: Missing staticfiles manifest entry for 'index.js'
========================================== 38 failed, 33 passed, 1 warning in 1.20s ======================================
```

Repeat with `DEBUG=True` and `DEBUG=False`.

**Comment by RealOrangeOne** [comment]
I believe this is fixed by #72 ?
